components/furnituremix/application/src/app/routes/shop/$folder[.]pdf.tsx

```diff
@@ -0,0 +1,46 @@
+import { LoaderFunction, Response } from '@remix-run/node';
+import ReactPDF from '@react-pdf/renderer';
+
+import { StoreFrontAwaretHttpCacheHeaderTagger } from '~/core-server/http-cache.server';
+import { getStoreFront } from '~/core-server/storefront.server';
+import { CrystallizeAPI } from '~/use-cases/crystallize';
+import { getHost } from '~/core-server/http-utils.server';
+
+import { Folder } from '~/core/components/pdf/folder';
+
+export const loader: LoaderFunction = async ({ request, params }) => {
```

Comment:
I wonder if we should group that logic, which is the same in the non-pdf thing.
@dhairyadwivedi maybe if you have time before me. Otherwise I will do that when I merge this tomorrow.

Reply:
I'll do that later today, merging this for now.